week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -1,64 +1,119 @@
+ *
+ * Iterate through the array of books passed into this function and insert each book to page as follows:
+ *   - Create a <ul> list and display each book details in <li> element.
+ *   - For each book, create a <p> element with the book title and author and append it to the page.
+ *   - Add an <img> after <p> element to each book that links to a URL of the book cover.
+ *     You should find an appropriate image to each book.
+ *   - Change the style of the book depending on whether you have read it (green) or not (red).*
+ *
+ * Find in the code what properties a book object has in the array.
+ *
+ * All of your HTML elements should go inside the Div tag with the id "content".
+ *
+ * The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com
+ */
 
-    const books = [
-        {
-            title: "The Design of Everyday Things",
-            author: "Don Norman",
-            alreadyRead: false
-        },
-        {
-            title: "The Most Human Human",
-            author: "Brian Christian",
-            alreadyRead: true
-        },
-        {
-            title: "The Pragmatic Programmer",
-            author: "Andrew Hunt",
-            alreadyRead: true
-        }
+function insertBooks(books) {
+  let content = document.querySelector("#content");
+  let unorderedList = document.createElement("ul");
+  content.appendChild(unorderedList);
+  for (let i = 0; i < books.length; i++) {
+    let liOfBook = document.createElement("li");
+    unorderedList.appendChild(liOfBook);
+    let bookList = document.createElement("p");
+    bookList.textContent = books[i].title + " - " + books[i].author;
+    liOfBook.appendChild(bookList);
+    let imgEl = document.createElement("img");
+    let images = [
+      "https://images-na.ssl-images-amazon.com/images/I/410RTQezHYL._SX326_BO1,204,203,200_.jpg",
+      "https://images-na.ssl-images-amazon.com/images/I/71HMyqG6MRL.jpg",
+      "https://images-na.ssl-images-amazon.com/images/I/418M2053aNL.jpg",
     ];
+    imgEl.src = images[i];
+    imgEl.width = "180";
+    imgEl.height = "280";
+    liOfBook.appendChild(imgEl);
 
-    Iterate through the array of books.
-    - For each book, create a <p> element with the book title and author and append it to the page.
-    - Use a <ul> and <li> to display the books.
-    - Add an <img> to each book that links to a URL of the book cover.
-    - Change the style of the book depending on whether you have read it (green) or not (red).
-
-    The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
-**/
-function exerciseThree(books) {
-  //Write your code in here
+    if (books[i].alreadyRead === true) {
+      liOfBook.style.backgroundColor = "green";
+    } else {
+      liOfBook.style.backgroundColor = "red";
+    }
+  }
+  unorderedList.style.display = "flex";
+  unorderedList.style.justifyContent = "space-around";
```

Comment:
Really nice solution and styling is almost right. I would only add some padding to `li` elements and remove list bullet points by adding this line of code:
``` unorderedList.style.listStyleType = "none";```
You could also declare an array with image URLs outside the loop. Images don't have links(anchor tags) to URLs with book covers as per the instruction but you did a good job🙂